src/components/shared/tests/__snapshots__/ManagedTree.spec.js.snap

```diff
@@ -225,9 +225,38 @@ exports[`ManagedTree sets expanded items 1`] = `
   getPath={[Function]}
   getRoots={[Function]}
   itemHeight={24}
-  onCollapse={[Function]}
-  onExpand={[Function]}
-  onFocus={[Function]}
+  onCollapse={[MockFunction]}
+  onExpand={
+    [MockFunction] {
+      "calls": Array [
```

Comment:
I'm thinking this new serialization of the calls array is what is causing my 1 failure in the tests for managed tree. 🤔

Reply:
Solved.